src/test/java/at/technikum/maintenancemonitor/service/MaintanenceMonitorServiceTest.java

```diff
@@ -56,14 +57,34 @@ public void downtimeTest() {
         } catch (InterruptedException e) {
             e.printStackTrace();
         }
```

Comment:
Timing is off since the service.start() gets called by the constructor now. 
Improve by resetting the values when calling service.start()